modules/virt-vm-wizard-fields-web.adoc

```diff
@@ -84,9 +84,16 @@ endif::[]
 
 |Flavor
 |Tiny, Small, Medium, Large, Custom
-|Presets that determine the amount of CPU and memory allocated to the virtual machine. The presets displayed for *Flavor* are determined by the operating system.
+|Presets the amount of CPU and memory in a virtual machine template with predefined values that are allocated to the virtual machine, depending on the operating system associated with that template.
 
-.3+|Workload Type
+If you choose a default template, you can override the `cpus` and `memsize` values in the template using custom values to create a custom template. Alternatively, you can create a custom template by modifying the `cpus` and `memsize` values in the *Details* tab on the *Workloads* -> *Virtualization* page.
+
+.3+a|Workload Type
+
+[NOTE]
+====
+If you choose the incorrect Workload Type, there could be performance or resource utilization issues (such as a slow UI).
```

Comment:
**Workload Type** if you are referring to the actual UI element, otherwise s/Workload Type/workload type